requirements.txt

```diff
@@ -1,4 +1,5 @@
-git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice]
+PyYAML>=3.0
 astral>=1.2
 ckuehl-upsidedown>=0.4
 psutil>=5.2
+git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice]
```

Comment:
We can remove the voice part. I don't think we will ever use it considering we are a self bot